### ISSUE TITLE
False Unreachable Code detected for EXEC CICS HANDLE

### ISSUE THREAD

**Issue by phaumer** [IBM/zopeneditor-about#52]
Extracted from here:
- https://github.com/IBM/zopeneditor-about/issues/18#issuecomment-615279300
- https://github.com/IBM/zopeneditor-about/issues/18#issuecomment-618093405

**Comment by oakmount66** [comment]
Hi Peter.

We see that exec cics syncpoint gives a similar false "Unreachable code".

![image](https://user-images.githubusercontent.com/60236192/98541024-156c0300-228f-11eb-9a93-96d38f867ac4.png)

![image](https://user-images.githubusercontent.com/60236192/98541074-26b50f80-228f-11eb-94a3-7f8a22623215.png)

Best regards, Lars.

**Comment by ian-burnett** [comment]
Similar circumstance for CICS labels. Sample program:

      *    Missing queue is correct path here
           exec cics handle condition qiderr(no-queue) end-exec.

      *    Read from the TSQ - all non-QIDERR errors will abend
           exec cics readq ts qname(tsq-name-control)
                              into(tsq-control-data)
                              length(ws-control-data-len)
                              item(1)
                              end-exec.

      *    Successful completion means we need to delete
           exec cics send text from(msg-tsq-exists)
                     erase freekb end-exec.
           goback.

      *    Correct to branch here due to missing TSQ
      *    >>> REPORTED INCORRECTLY AS UNREACHABLE CODE <<<
       no-queue.

**Comment by ian-burnett** [comment]
Stumbled upon this again, but can now add more diagnostics:

```cobol
       PROCESS CICS

       identification division.
       program-id.              NOREACH.

       data division.
       working-storage section.

       procedure division using dfheiblk.

       main-processing section.

           exec cics handle condition error(error-handle) end-exec.

           exec cics handle condition duprec(DUPREC-HANDLE) end-exec.

           go to all-done.

       error-handle.         *> Incorrectly flagged unreachable
           go to all-done.

       duprec-handle.
           go to all-done.

       all-done.
           goback.
```

Works OK if the statement label in the EXEC CICS command is in uppercase, but not if it is lower case.

Note the case of the label itself makes no difference.

**Comment by sdaimwood** [comment]
These bugs were fixed in version 4.3.0, thank you for your patience while we worked on a fix.
